### PR TITLE
Reduce audio message size

### PR DESCRIPTION
**Pull request by johnxnguyen**
## Problem
Audio messages are being recorded at a relatively high sample rate (44.1khz, the same used for CDs) with a variable bit rate that averages around 200kbps. This leads to large file sizes and impacts upload/download performance.

## Solution
Lowering the sample rate to only 32khz generates files with an average bitrate of 47kbps, effectively reducing the file size to a quarter of the original, while at the same time maintaining acceptable audio quality for voice recorded messages.

### CODE REVIEW

File: Wire-iOS/Sources/AppRootViewController.swift [hunk 1]
```diff
@@ -54,7 +54,7 @@ class AppRootViewController : UIViewController {
         
         let isCallkitEnabled = !Settings.shared().disableCallKit
         var isCallkitSupported = false
-        if #available(iOS 10, *), TARGET_OS_SIMULATOR != 0 {
```
Comment (mikeger) [inline Wire-iOS/Sources/AppRootViewController.swift]: Not sure if this should be removed, I think the simulator does not support the CallKit.

**Comment by mikeger** [comment]
@johnxnguyen I am not sure,  I think the PR contains some unnecessary changes for AVS and SE versions, may be those can be separated into another PR for consistency.

File: Wire-iOS/Sources/AppRootViewController.swift (hunk 1 above)
Comment (typfel) [reply Wire-iOS/Sources/AppRootViewController.swift]: We need to find another way to detect. This compiled with an error against a device.

**Comment by johnxnguyen** [comment]
@mikeger do you mean making a PR for only the component bumps?

File: Wire-iOS/Sources/AppRootViewController.swift (hunk 1 above)
Comment (johnxnguyen) [reply Wire-iOS/Sources/AppRootViewController.swift]: ok I'll take a look :)

Comment (mikeger) [reply Wire-iOS/Sources/AppRootViewController.swift]: We can do `#if arch(i386) || arch(x86_64)`

File: Wire-iOS/Sources/AppStateController.swift [hunk 2]
```diff
@@ -105,6 +105,10 @@ class AppStateController : NSObject {
 
 extension AppStateController : SessionManagerDelegate {
     
+    func sessionManagerWillSuspendSession() {
```
Comment (typfel) [inline Wire-iOS/Sources/AppStateController.swift]: Don't don't know why this didn't conflict. But you should take the implementation from develop.

**Comment by mikeger** [comment]
@johnxnguyen would be cool, just for the consistency, so later in the commit tree we would not have the commit to reduce the audio message size that also bumps the AVS and updates some internal behavior, certainly if AVS update is not necessary for the audio message size reduction,

**Comment by johnxnguyen** [comment]
@mikeger yeah that makes sense, I just ran `carthage update` to get the new versions.

File: Wire-iOS/Sources/AppStateController.swift [hunk 3]
```diff
@@ -77,7 +77,7 @@ class AppStateController : NSObject {
             return .blacklisted
         }
         
-        if (isSuspended) {
+        if isSuspended {
```
Comment (typfel) [inline Wire-iOS/Sources/AppStateController.swift]: 🙌

**Pull request merged**